helpers/github.ts

```diff
@@ -7,6 +7,16 @@ import { writeFile } from "fs/promises";
 import twitter from "./twitter";
 import { TwitterMap } from "..";
 
+// Usually
+export const DEVPOOL_OWNER_NAME = process.env.DEVPOOL_OWNER_NAME;
+export const DEVPOOL_REPO_NAME = process.env.DEVPOOL_REPO_NAME;
+export const IS_RFC = process.env.RFC == "1";
```

Comment:
I would JSON.parse this environment variable for a boolean. Doesn't make sense to me why you would use a number in place of a boolean.

Reply:
done